src/WebJobs.Script/Description/Proxy/ProxyFunctionInvoker.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.AppService.Proxy.Client.Contract;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    internal class ProxyFunctionInvoker : FunctionInvokerBase
+    {
+        private IProxyClient _proxyClient;
+
+        public ProxyFunctionInvoker(ScriptHost host, FunctionMetadata functionMetadata, IProxyClient proxyClient) : base(host, functionMetadata)
+        {
+            _proxyClient = proxyClient;
+        }
+
+        protected override async Task InvokeCore(object[] parameters, FunctionInvocationContext context)
+        {
+            await _proxyClient.CallAsync(parameters, null, context.Logger);
```

Comment:
>_proxyClient.CallAsync(parameters, [](start = 18, length = 34)

Can we give proxyClient a strong signature? IE, it it's really Task  Invoke(HttpRequestMessage request), then let's just make it that.  In general, we want strong signature on the cross-team contract  to reduce chances of breaking.